.github/workflows/release_version.yml

```diff
@@ -83,27 +76,44 @@ jobs:
           else
             export REL_VER=`git describe --tags --always`
           fi
+          if [[ $REL_VER == tikv-br* ]];then
```

Comment:
How about name tags as `br`/`cdc`/`gc-worker` and concat `PROD_NAME=tikv-$REL_VER` ? Then we don't need the condition judgements here.
It is ok to me that gc-worker is named as `tikv-gc-worker`.

Reply:
Besides, I think it would be better to limit the `$REL_VER` as one of `cdc`, `br`, `gc-worker`, to avoid delivering an unexpected package.

Reply:
OK, agreed. The only concern is the name `tikv-gc-worker` previously. `tikv-gc-worker`  also seems reasonable.

Reply:
Validition check for tag is added, PTAL~